src/main/java/uk/gov/companieshouse/api/accounts/controller/ProfitLossController.java

```diff
@@ -0,0 +1,131 @@
+package uk.gov.companieshouse.api.accounts.controller;
+
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.MediaType;
+import org.springframework.http.ResponseEntity;
+import org.springframework.validation.BindingResult;
+import org.springframework.web.bind.annotation.DeleteMapping;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.PathVariable;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.PutMapping;
+import org.springframework.web.bind.annotation.RequestBody;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RestController;
+import uk.gov.companieshouse.api.accounts.AttributeName;
+import uk.gov.companieshouse.api.accounts.exception.DataException;
+import uk.gov.companieshouse.api.accounts.links.SmallFullLinkType;
+import uk.gov.companieshouse.api.accounts.model.rest.SmallFull;
+import uk.gov.companieshouse.api.accounts.model.rest.profitloss.ProfitLoss;
+import uk.gov.companieshouse.api.accounts.model.validation.Errors;
+import uk.gov.companieshouse.api.accounts.service.impl.ProfitLossService;
+import uk.gov.companieshouse.api.accounts.service.response.ResponseObject;
+import uk.gov.companieshouse.api.accounts.utility.ApiResponseMapper;
+import uk.gov.companieshouse.api.accounts.utility.ErrorMapper;
+import uk.gov.companieshouse.api.accounts.utility.LoggingHelper;
+import uk.gov.companieshouse.api.model.transaction.Transaction;
+
+import javax.servlet.http.HttpServletRequest;
+import javax.validation.Valid;
+
+@RestController
+@RequestMapping(value = "/transactions/{transactionId}/company-accounts/{companyAccountId}/small-full/notes/profit-and-loss", produces = MediaType.APPLICATION_JSON_VALUE)
+public class ProfitLossController {
+
+    @Autowired
+    private ProfitLossService profitLossService;
+
+    @Autowired
+    private ApiResponseMapper apiResponseMapper;
+
+    @Autowired
+    private ErrorMapper errorMapper;
+
+    @PostMapping
+    public ResponseEntity create(@Valid @RequestBody ProfitLoss profitLoss,
```

Comment:
Something I didn't consider, the `ProfitLoss` object will need to be refactored to become `ProfitAndLoss`. When Jackson serializes an object, the object name is snake-cased and prefixed with a dollar to create the prefix of the json path.

As the object is currently named, Spring's binding result errors will result in json of the form:
```
$.profit_loss.<path_of_field_in_error>
```

Renaming this object to Profit**And**Loss will correctly yield validation errors of the format:
```
$.profit_and_loss.<path_of_field_in_error>
```